packages/shorebird_cli/lib/src/shorebird_flutter.dart

```diff
@@ -62,7 +71,24 @@ class ShorebirdFlutter {
       rethrow;
     }
 
-    installProgress.complete();
+    final precacheProgress = logger.progress(
+      'Running ${lightCyan.wrap('flutter precache')}',
+    );
+
+    try {
+      await process.run(
```

Comment:
What about initial install? Should we also do this as part of the install script for consistency?

Reply:
Good idea, filed https://github.com/shorebirdtech/shorebird/issues/2093